src/python.ts

```diff
@@ -70,7 +66,7 @@ export async function initializePython(disposables: Disposable[]): Promise<void>
 
         if (api) {
             disposables.push(
-                api.environments.onDidChangeActiveEnvironmentPath((event) => {
+                api.environments.onDidChangeActiveEnvironmentPath((event: any) => {
```

Comment:
I don't know about the typing there and below but is there a way to remove any?

Reply:
I tried to find the exact typing, but my esLinter didn't make me much wiser. Don't know if it's worth the effort to dig deeper on that part right now.